common/source/docs/common-rc-transmitter-flight-mode-configuration.rst

```diff
@@ -80,7 +80,8 @@ The flight mode channel is the input radio channel that ArduPilot
 monitors for mode changes.
 
 [site wiki="copter"]
-On Copter this is always channel 5.
+On Copter this is configurable using the :ref:`FLTMODE_CH <plane:FLTMODE_CH>`
```

Comment:
```suggestion
On Copter this is configurable using the :ref:`FLTMODE_CH <FLTMODE_CH>`
```
its in the copter wiki parameters page now...no need for the redirection to plane....